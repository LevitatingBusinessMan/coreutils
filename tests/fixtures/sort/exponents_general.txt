100E6

50e10
+100000

10000K78
10E


1000EDKLD


100E6

50e10
+100000

10e10e10e10
5.5.5.5
