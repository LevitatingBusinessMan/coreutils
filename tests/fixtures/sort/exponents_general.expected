







5.5.5.5
10E
1000EDKLD
10000K78
+100000
+100000
100E6
100E6
10e10e10e10
50e10
50e10
